spikeextractors/extractors/neuroscopeextractors/neuroscopeextractors.py

```diff
@@ -33,6 +32,21 @@ def get_shank_files(folder_path: Path, suffix: str):
         and re.search(r"\d+$", f.name) is not None and len(f.suffixes) == 2
     ]
 
+def find_xml_file_path(folder_path: PathType):
+    xml_files = [f for f in folder_path.iterdir() if f.is_file() if f.suffix == ".xml"]
+    assert any(xml_files), "No .xml files found in the folder_path."
+    assert len(xml_files) == 1, "More than one .xml file found in the folder_path! Specify xml_file_path."
+    xml_file_path = xml_files[0]
+    return xml_file_path
+
+def handle_xml_file_path(folder_path: PathType, initial_xml_file_path: PathType):
+    if initial_xml_file_path is None:
+        xml_file_path = find_xml_file_path(folder_path=folder_path)
+    else:
+        assert Path(initial_xml_file_path).is_file(), f".xml file ({initial_xml_file_path}) not found!"
+        xml_file_path = initial_xml_file_path
+    return xml_file_path
```

Comment:
@CodyCBakerPhD and @alejoe91 this is minor, but I'd prefer an early return pattern in these types of situations, i.e.

```python
def handle_xml_file_path(folder_path: PathType, initial_xml_file_path: PathType):
    if initial_xml_file_path is None:
        return find_xml_file_path(folder_path=folder_path)
    assert Path(initial_xml_file_path).is_file(), f".xml file ({initial_xml_file_path}) not found!"
    return initial_xml_file_path
```